app/controllers/subjects_controller.rb

```diff
@@ -1,6 +1,6 @@
 class SubjectsController < ApplicationController
   def index
-    search = Search.new nil, { facets: %w(subject), page_size: 0, facet_size: 500 }
+    search = Search.new nil, { facets: %w(subject), page_size: 0, facet_size: 240 }
```

Comment:
This powers the subjects page (dp.la/subjects), correct?  Is that in danger of the sort of malicious attacks we've been seeing?  If not, it might be worth keeping the facet size high in this instance, since the purpose of the page seems to be to list all our subjects for browsing purposes.